compiler/front/cli_reporter.nim

```diff
@@ -4179,6 +4190,25 @@ func astDiagToLegacyReport(diag: PAstDiag): Report {.inline.} =
       kind: rsemCannotCastToNonConcrete,
       ast: diag.wrongNode,
       typ: diag.wrongType)
+  of adSemFoldRangeCheckForLiteralConversionFailed:
+    semRep = SemReport(
+      location: some diag.location,
+      reportInst: diag.instLoc.toReportLineInfo,
+      kind: rsemCantConvertLiteralToRange,
+      ast: diag.wrongNode,              # xxx: ignores `diag.wrongNode`
```

Comment:
What is the comment referring to?

Reply:
Ah, the report ignores the `ast` field, which is set to `diag.wrongNode`.